plugin/storage/factory_config.go

```diff
@@ -29,13 +30,21 @@ const (
 	DependencyStorageTypeEnvVar = "DEPENDENCY_STORAGE_TYPE"
 
 	spanStorageFlag = "--span-storage.type"
+
+	// DownSamplingRatio is the name of the env var that defines the ratio of spans we would drop
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences let's try to follow golang conventions for comments. We should end everything with a period.